coreos-koji-tagger/coreos_koji_tagger.py

```diff
@@ -44,7 +44,7 @@
 
 # XXX: should be in config file
 DEFAULT_GITHUB_REPO_FULLNAME = 'coreos/fedora-coreos-config'
-DEFAULT_GITHUB_REPO_BRANCHES = 'refs/heads/testing-devel refs/heads/next-devel refs/heads/rawhide'
+DEFAULT_GITHUB_REPO_BRANCHES = 'refs/heads/testing-devel refs/heads/next-devel refs/heads/rawhide  refs/heads/branched'
```

Comment:
Hmm, maybe at this point we should just trim the `refs/heads/` and account for that in the (AFAICT) one spot that cares in `process_message`. But meh... obviously fine as is too!